boards/arm/thingy52_nrf52832/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 zephyr_library()
-zephyr_library_sources(board.c)
+zephyr_library_sources_ifdef(CONFIG_GPIO board.c)
```

Comment:
We need to avoid an empty library.

Put all this code under if(CONFIG_GPIO) ?

Reply:
Ok, applied the change.